### PR TITLE
MARKET-1576 Transitions

### PR DESCRIPTION
**Pull request by StasivPavlo**

### CODE REVIEW

File: components/bl-transitions-component/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Transition is a component of the Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). Transition allows you to add a smooth component mounting animation.
+
+The component based on external [Transition](https://mui.com/material-ui/transitions/).
+
+## Properties
+
+| Property          | Type                                                             | Default Value  | Logic | Data Binding | UI Setting | Description                                     |
+|-------------------|------------------------------------------------------------------|----------------|-------|--------------|------------|-------------------------------------------------|
+| variants          | Select ["collapse-top", "collapse-left", "fade", "grow", "zoom"] | "collapse-top" |       | NO           | YES        | Allows to determine variant of transition       |
+| animationDuration | Number                                                           | 300            |       | NO           | YES        | Allows to determine duration of transition (ms) |
```
Comment (addKiwi) [inline components/bl-transitions-component/README.md:12]: I think we should use labels like this: `Variants`, `Animation Duration`. In general, we make it for customers, not for developers.

File: components/bl-transitions-component/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Transition is a component of the Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). Transition allows you to add a smooth component mounting animation.
+
+The component based on external [Transition](https://mui.com/material-ui/transitions/).
+
+## Properties
+
+| Property          | Type                                                             | Default Value  | Logic | Data Binding | UI Setting | Description                                     |
+|-------------------|------------------------------------------------------------------|----------------|-------|--------------|------------|-------------------------------------------------|
+| variants          | Select ["collapse-top", "collapse-left", "fade", "grow", "zoom"] | "collapse-top" |       | NO           | YES        | Allows to determine variant of transition       |
```
Comment (addKiwi) [inline components/bl-transitions-component/README.md:11]:
```suggestion
| variants          | Select  <br/>["collapse-top", "collapse-left", "fade", "grow", "zoom"] | "collapse-top" |       | NO           | YES        | Allows to determine variant of transition       |
```

File: components/bl-transitions-component/README.md (hunk 1 above)
Comment (addKiwi) [inline components/bl-transitions-component/README.md:12]:
```suggestion
| variants          | Select <br/>["collapse-top", "collapse-left", "fade", "grow", "zoom"] | "collapse-top" |       | NO           | YES        | Allows to determine variant of transition       |
| animationDuration | Number                                                           | 300            |       | NO           | YES        | Allows to determine duration of transition (ms) |
```
I think we should use labels like this: Variants, Animation Duration. In general, we make it for customers, not for developers.

File: components/bl-transitions-component/component.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    }
+  ],
+  "actions": [],
+  "pods": {
+    "d562eaea8fcc85660b2f4124a92456d2": {
+      "id": "transitionsContainer",
+      "dataContainer": true
+    }
+  },
+  "blockUI": "<div data-module-type=\"system\" data-module-id=\"block\" data-display id=\"transitionsContainer\" data-uid=\"d562eaea8fcc85660b2f4124a92456d2\" data-pod-container=\"true\" data-data-container=\"true\" style=\"display:flex;flex-shrink:0;flex-direction:column;padding:10px;\"></div>"
```
Comment (addKiwi) [inline components/bl-transitions-component/component.json:78]: Redundant

**Comment by v-excelsior** [comment]
LGTM

**Review by v-excelsior — APPROVED** [review]

**Pull request merged**